regions/io/core.py

```diff
@@ -221,3 +371,120 @@ def to_region(self):
 
     def _raise_error(self, msg):
         raise self.error[self.format_type](msg)
+
+    def to_crtf(self):
+        """
+        Makes shape CRTF compatible.
+        """
+        self.format_type = 'CRTF'
+
+    def to_ds9(self):
+        """
+        Makes shape DS9 compatible
+        """
+        self.format_type = 'DS9'
+
+    def _validate(self):
+        """
+        Checks whether all the attributes of this object is valid according to the given format.
+        """
+        if self.format_type not in ['CRTF', 'DS9']:
+            raise ValueError("{0} is not available as io".format(self.format_type))
+        if self.region_type not in regions_attributes.keys():
+            raise ValueError("{0} is not a valid region type in this package".format(self.region_type, self.format_type))
+        if self.coordsys not in valid_coordsys[self.format_type]:
+            raise ValueError("{0} is not a valid coordinate reference frame in astropy".
+                             format(self.coordsys, self.format_type))
+
+
+def to_shape_list(region_list, format_type='DS9', coordinate_system='fk5'):
+
+    shape_list = ShapeList()
+
+    for region in region_list:
+
+        coord = []
+        reg_type = str(type(region)).split(".")[2]
+
+        for val in regions_attributes[reg_type]:
+            coord.append(getattr(region, val))
+
+        if reg_type == 'polygon':
+            coord = [x for x in region.vertices]
+
+        coordsys = 'image'
+        if isinstance(region, SkyRegion):
+            coordsys = coord[0].name
+
+        coordsys = coordinate_system or coordsys
```

Comment:
this line and the previous 3 lines could probably be made a little clearer and more concise.  e.g.:
```
if coordinate_system:
    coordsys = coordinate_system
else:
    if isinstance(region, SkyRegion):
        coordsys = coord[0].name
     else:
         coordsys='image'
```
I believe that logic is the same as what you've written, but it's more linear

Reply:
Thanks! this is much simpler.